docs/Testing-docs/Test-workflows-automation.md

```diff
@@ -1,12 +1,13 @@
 # Automating test workflows in the PR checks.
 Test workflows for actions can be automated in the action repo so that whenever a new PR is raised to __master__ or __releases/*__  branches these workflows evaluate on the branch from which PR is raised. <br>
 
-This process of automated testing enables one to run tests on PRs from a branch in a repo and also  PRs from a forked repo. Post reviewing the PR for security vulnerabilities, the author(s) / admin(s) can set a label as  ```safe to run test``` on the PR which will trigger this workflow automatically. This will ensure that the new PR is not breaking any existing flow. Once the checks are successful and are not breaking any existing scenarios, the PR will be merged subsequently.
+This process of automated testing enables one to run tests on PRs from a branch in a repo and also  PRs from a forked repo. Inorder to ensure the safety of secrets which are used by the pr-check workflows and to prevent pwn requests, the pr-check workflow and secrets should be a part of a [github environment](https://docs.github.com/en/actions/reference/environments) and set appropriate approval policy for triggering this workflow on a new PR. <br>
```

Comment:
`prevent pwn request`
Seems like a typo

Reply:
No it exists https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
Interesting!